### DISCUSSION TITLE
running-kubernetes-and-the-dashboard-with-docker-desktop/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#92] [Announcements]
# running-kubernetes-and-the-dashboard-with-docker-desktop/

In this post I show how to enable Kubernetes with Docker Desktop, how to install and patch the Kubernetes dashboard, and how to enable CPU and memory graphs.

https://andrewlock.net/running-kubernetes-and-the-dashboard-with-docker-desktop/

**Comment by MxMatrixDemon** [comment]
Hi Andrew,
I still got a "nodes is forbidden: User "system:serviceaccount:kubernetes-dashboard:kubernetes-dashboard" cannot list resource "nodes" in API group "" at the cluster scope" after completing your instructions. Any suggestions how to solve this?
Regards,
Guus.

**Reply by a11smiles** [reply]
So did I. As the error says, it's a permissions issue. There's a two-step process for fixing this. Execute the following two commands:

```bash
$ kubectl delete clusterrolebinding serviceaccount-cluster-admin
$ kubectl create clusterrolebinding serviceaccount-cluster-admin --clusterrole=cluster-admin --user=system:serviceaccount:kubernetes-dashboard:kubernetes-dashboard
```

The first command may tell you that the user doesn't exist. That's fine, just run the command to make sure. It's the second command that gives you the permissions. Once you run the second command, restart `kubectl proxy` and refresh your browser. You should be good to go.

**Comment by venkatamatta** [comment]
Hi Andrew,

Thankyou for your blog.

I am bale to install the dashboard successfully 

i am able to view only the default namespaces but i have the multiple namespaces, Please let me did i miss any configurations .

Regards,
Venkat

**Reply by venkatamatta** [reply]
its resolved and i created the admin-user and cluster role binding account to get the token

**Reply by venkatamatta** [reply]
But for the metric server i am getting below error

Readiness probe failed: HTTP probe failed with statuscode: 500

[a comment by an automated account is not shown]

**Comment by ckadarla** [comment]
Can we access the k8s dashboard without proxy I mean to say that using ingress
